security/pfSense-pkg-acme/files/usr/local/www/acme/acme_accountkeys.php

```diff
@@ -0,0 +1,281 @@
+include("head.inc");
+if ($input_errors) {
+	print_input_errors($input_errors);
+}
+if ($savemsg) {
+	print_info_box($savemsg);
+}
+
+/*$display_apply = file_exists($d_acmeconfdirty_path) ? "" : "none";
+echo "<div id='showapplysettings' style='display: {$display_apply};'>";
+print_apply_box(sprintf(gettext("The configuration has been changed.%sYou must apply the changes in order for them to take effect."), "<br/>"));
+echo "</div>";
+*/
+?>
+<div id="renewoutputbox" class="alert alert-success clearfix hidden" role="alert">
+	<button type="button" class="close" data-dismiss="alert" aria-label="Close">
+		<span aria-hidden="true">×</span>
+	</button>
+	<div id="renewoutput" class="pull-left">
+	</div>
+</div>
+
+<?php
+display_top_tabs_active($acme_tab_array['acme'], "accountkeys");
+?>
+<form action="acme_accountkeys.php" method="post">
+	<div class="panel panel-default">
+		<div class="panel-heading">
+			<h2 class="panel-title">Account keys</h2>
+		</div>
+		<div id="mainarea" class="table-responsive panel-body">
+			<table class="table table-hover table-striped table-condensed">
+				<thead>
+					<tr>
+						<th></th>
+						<th width="30%">Name</th>
+						<th width="20%">Description</th>
+						<th>CA</th>
+						<th>Actions</th>
+					</tr>
+				</thead>
+				<tbody class="user-entries">
+<?php
+		foreach ($a_accountkeys as $accountkey) {
+			$accountname = $accountkey['name'];
+			?>
+			<tr id="fr<?=$accountname;?>" <?=$display?> onClick="fr_toggle('<?=$accountname;?>')" ondblclick="document.location='acme_accountkeys_edit.php?id=<?=$accountname;?>';">
+				<td>
+					<input type="checkbox" id="frc<?=$accountname;?>" onClick="fr_toggle('<?=$accountname;?>')" name="rule[]" value="<?=$accountname;?>"/>
+					<a class="fa fa-anchor" id="Xmove_<?=$accountname?>" title="<?=gettext("Move checked entries to here")?>"></a>
+				</td>
+			  <td>
+				<?=$accountkey['name'];?>
+			  </td>
+			  <td>
+				<?=$accountkey['desc'];?>
+			  </td>
+			  <td>
+				<?=$accountkey['acmeserver'];?>
+			  </td>
+			  <td class="action-icons">
+				<button style="display: none;" class="btn btn-default btn-xs" type="submit" id="move_<?=$accountname?>" name="move_<?=$accountname?>" value="move_<?=$accountname?>"></button>
+				<a href="acme_accountkeys_edit.php?id=<?=$accountname;?>">
+					<?=acmeicon("edit", gettext("edit"))?>
+				</a>
+				<a href="acme_accountkeys.php?act=del&amp;id=<?=$accountname;?>" onclick="return confirm('Do you really want to delete this entry?')">
+					<?=acmeicon("delete", gettext("delete"))?>
+				</a>
+				<a href="acme_accountkeys_edit.php?dup=<?=$accountname;?>">
+					<?=acmeicon("clone", gettext("clone"))?>
+				</a>
+			  </td>
+			</tr><?php
+		}
+?>				
+				</tbody>
+			</table>
+		</div>
+	</div>
+	<nav class="action-buttons">
+		<a href="acme_accountkeys_edit.php" role="button" class="btn btn-sm btn-success" title="<?=gettext('Add backend to the end of the list')?>">
+			<i class="fa fa-level-down icon-embed-btn"></i>
```

Comment:
Single add buttons should use fa-plus, not fa-level-down. Only use fa-level-down when the page supports adding to the top or bottom of a list (For example, when there is also an fa-level-up add button).